modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-values-factory/src/main/java/com/liferay/dynamic/data/mapping/form/values/factory/internal/DDMFormValuesFactoryImpl.java

```diff
@@ -137,18 +144,30 @@ private void _checkDDMFormFieldParameterNames(
 				ddmFormFieldParameterNames.add(
 					defaultDDMFormFieldParameterName);
 
-				_checkDDMFormFieldParameterNames(
-					ddmFormField.getNestedDDMFormFields(), StringPool.BLANK,
-					ddmFormFieldParameterNames);
+				if (!fullDDMFormFields.containsKey(ddmFormField.getName())) {
```

Comment:
Do we need to add this condition `!fullDDMFormFields.containsKey(ddmFormField.getName())` ?
Because there is the opposite condition in the lines 125-128